src/josh.js

```diff
@@ -62,7 +62,7 @@ class Josh {
   }
   //Adding css to DOM
   addCss(targetElm) {
-    if (targetElm.length > 0) {
+    if (targetElm.length >= 0) {
```

Comment:
Isn't what you're trying to figure out here if `targetElm` is an array?
In that case this might be more appropriate
```suggestion
    if (Array.isArray(targetElm.length)) {
```

Reply:
Thank you. No, it's not array. it's [NodeList](https://developer.mozilla.org/ja/docs/Web/API/NodeList). But your suggestion is good, I decide to use `instanceof` for this.

Reply:
Done by force push.